docs/content/en/docs/getting-started/optional/registrymirror.md

```diff
@@ -152,8 +151,7 @@ isovalent
 cilium-chart
 ```
 
-For example, if a registry is available at `private-registry.local`, then the following 
-projects will have to be created:
+For example, if a registry is available at `private-registry.local`, then the following projects will have to be created:
```

Comment:
There is nothing following this statement? Is there supposed to be an example here that's missing?

Reply:
is it wrapped by github UI?